docs/source/gdpr/credit.rst

```diff
@@ -4,33 +4,53 @@
 Background
 ==========
 
-To facilitate neuroimaging data sharing, a template data sharing consent form was generated by pooling forms from many places and being reviewed by ethic experts and lawyers. This consent doesn't comply with GDPR, and work has been done during the 2019 OHBM Hackathon toward updating the consent: see `this google doc
-<https://docs.google.com/document/d/1Mfbl4DZAw7MRPjSxIiM5sfYU4gX-pcghgj5M1qb84jg/edit>`_
+To facilitate neuroimaging data sharing, a template data sharing consent form was generated by pooling forms from many places and being reviewed by ethic experts and lawyers, resulting in a 1st draft of open consent around 2015. This consent doesn't however comply with the European General Data Protection Regulation (`GDPR<https://gdpr-info.eu>`_), and work has been done in June 2019 during OHBM Hackathon toward updating the consent: see `this google doc
+<https://docs.google.com/document/d/1Mfbl4DZAw7MRPjSxIiM5sfYU4gX-pcghgj5M1qb84jg/edit>`_ Following up on this work, the `GliMR<https://glimr.eu/>`_ `COST<https://www.cost.eu/>`_ action joined the project and organized a workshop in November 2019 to create a new consent form.
```

Comment:
There must be a space before `<` in links. Have a look at rendered version https://open-brain-consent--67.org.readthedocs.build/en/67/gdpr/credit.html#id1

The same for other links eg in the bottom of the page

May be use didn't instead of doesn't for older draft?